content/blog/TinaCMS-New-Public-Roadmap.mdx

```diff
@@ -7,14 +7,14 @@ author: James O'Halloran
 
 TinaCMS now has a public roadmap where you can follow along with high-level features on our radar.&#x20;
 
-![Public Roadmap](https://res.cloudinary.com/forestry-demo/image/upload/v1669232450/blog-media/roadmap/Screen_Shot_2022-11-23_at_2.22.24_PM.png)
+![Public Roadmap](/img/blog/tinacms-new-public-roadmap/public-roadmap.png)
 
 The roadmap is reserved for items in which we have a high degree of confidence that they'll fall into a given timeframe. Each roadmap item will link to a GitHub discussion, where you can comment on features, or subscribe for updates.\
 \
 If you'd like to see something make its way into the roadmap, start by creating a feature request in our [GitHub Discussions](https://github.com/tinacms/tinacms/discussions "GitHub Discussions").
 
 Under the hood, the roadmap is also managed with Tina! So we'll be jumping it and keeping this page up to date regularly as plans change and as progress is made. \
 \
```

Comment:
**issue**: Though this line was not changed, the edit to this file now means that the slash is being displayed.

![image](https://github.com/user-attachments/assets/5ee68efb-a6e8-4463-9e73-de999355f20c)
**Figure: Slash added to blog post**

Reply:
Fixed